nifi-extension-bundles/nifi-kafka-bundle/nifi-kafka-3-service/src/main/java/org/apache/nifi/kafka/service/consumer/Kafka3ConsumerService.java

```diff
@@ -185,14 +185,20 @@ public ByteRecord next() {
                 recordHeaders.add(recordHeader);
             });
 
+            // NIFI-14122: Support Kafka tombstones
```

Comment:
Code comments like this (with JIRA numbers) in the production code are generally disfavored (though they do exist).  Maybe just:

`Support Kafka tombstones`